presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveDynamicPartitionPruning.java

```diff
@@ -166,7 +170,7 @@ public void testJoinWithNonSelectiveBuildSide()
     }
 
     @Test(timeOut = 30_000)
-    public void testJoinLargeBuildSideNoDynamicFiltering()
+    public void testJoinLargeBuildSideDynamicFiltering()
```

Comment:
Maybe call it `testJoinLargeBuildSideRangeDynamicFiltering`
Also, we are missing test case for "too" large build side for DFs to be collected. It would be good to restore it, but it's not critical

Reply:
agreed, I'll add the too-large-no-df test as well in a different PR